File: C#/05-Operadores Lógicos/Logicos.cs
using System;

class Logico
{
    static void Main(string[] args)
    {
    //Os operadores lógicos em C# são:
    // && e = teste retorna VERDADEIRO se e somente se as duas condições forem verdadeiras
    // || ou = teste retorna VERDADEIRO se pelo menos uma das condições for verdadeira
    // ! não = Inverte o resultado do teste lógico

        double k = 1.2;
        double l = 1.5;
        double m = 1.2;
        bool teste;
    
    //Teste com operador E
        teste = (k == m && k <= m);
        Console.WriteLine(teste); //Retorna true pois as duas expressões são verdadeiras
       
        teste = (k == l && k <= m);
        Console.WriteLine(teste); //Retorna false pois apenas uma expressão é verdadeira

    //Teste com operador OU
        teste = (k < l || k != m);    
        Console.WriteLine(teste); //Retorna true pois pelo menos uma expressão é verdadeira

        teste = (k == l || k > m);
        Console.WriteLine(teste); //Retorna false pois as duas expressões são falsas

    //Teste com operador NÂO
        teste = !(k == m);
        Console.WriteLine(teste); //Retorna false pois resultado do teste foi invertido   

        Console.ReadLine();
        
    }
}

File: PHP/05-Operadores Lógicos/Logicos.php
<?php

//Operadores lógicos podem testar multiplas condições
//Os operadores lógicos em php são : &&[E] ||[OU] ![NÃO]

$x = 4;
$y = 6;
$z = 9;

echo"1 - VERDADEIRO / [NDA] - FALSO";
echo'<br/>';

//O operador && só retorna verdadeiro se todas condições
//forem verdadeiras
echo "Operador &&";
echo'<br/>';
echo $x ."<" . $y . " E ". $y . "<" . $z . " = ";
echo'<br/>';
echo ($x < $y) && ($y < $z); //v e v = v
echo'<br/>';

echo $x .">" . $y . " E ". $y . "<" . $z . " = ";
echo'<br/>';
echo ($x > $y) && ($y < $z); //f e v = f
echo'<br/>';

echo $x ."<=" . $y . " E ". $y . ">=" . $z . " = ";
echo'<br/>';
echo ($x <= $y) && ($y >= $z); //v e f = f
echo'<br/>';

echo $x ."==" . $y . " E ". $y . ">=" . $z . " = ";
echo'<br/>';
echo ($x == $y) && ($y >= $z); //f e f = f
echo'<br/>';

//O operador || só retorna falso se todas condições
//forem falsas
echo "Operador ||";
echo'<br/>';

echo $x ."<" . $y . " OU ". $y . "<" . $z . " = ";
echo'<br/>';
echo ($x < $y) || ($y < $z); // v ou v = v
echo'<br/>';

echo $x .">" . $y . " OU ". $y . "<" . $z . " = ";
echo'<br/>';
echo ($x > $y) || ($y < $z); // f ou v = v
echo'<br/>';

echo $x ."<=" . $y . " OU ". $y . ">=" . $z . " = ";
echo'<br/>';
echo ($x <= $y) || ($y >= $z); // v ou f = v
echo'<br/>';

echo $x ."==" . $y . " OU ". $y . ">=" . $z . " = ";
echo'<br/>';
echo ($x == $y) || ($y >= $z); // f ou f = f
echo'<br/>';

//O operador ! inverte o resultado do teste
echo "Operador !";
echo'<br/>';

echo "NAO " . $x ."<" . $y . " = ";
echo'<br/>';
echo !($x < $y); // não v = f
echo'<br/>';

echo "NAO " . $y .">" . $z . " = ";
echo'<br/>';
echo !($y > $z); // não f = v
echo'<br/>';

?>